core/mocks/generated/trezorui2.pyi

```diff
@@ -178,6 +178,7 @@ def show_info(
     title: str,
     description: str = "",
     time_ms: int = 0,
+    centered: bool = False,
```

Comment:
when do we want this to be false and when true? can you show examples?

Reply:
Normally it is false, putting that to true in this screen:
![image](https://github.com/trezor/trezor-firmware/assets/42543243/9106230d-cf50-4966-9336-e9545f7d62e5)